cms/plugin_base.py

```diff
@@ -386,22 +391,23 @@ def get_child_plugin_candidates(cls, slot, page):
         from cms.plugin_pool import plugin_pool
         return plugin_pool.get_all_plugins()
 
-    def get_child_classes(self, slot, page):
+    @classmethod
+    def get_child_classes(cls, slot, page):
```

Comment:
The door was already closed.
A `CMSPluginBase` instance was never guaranteed, and usually it would be created on the fly without any uniqueness to it meaning that there really wouldn't be a difference between instances of the same plugin class except for their memory id.

I'm aiming for decoupling more of these methods from the instance because it's very confusing and inconsistent that for some we need class methods and for other instance methods.